pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/PercentileTDigestAggregationFunction.java

```diff
@@ -38,21 +38,32 @@
 public class PercentileTDigestAggregationFunction extends BaseSingleInputAggregationFunction<TDigest, Double> {
```

Comment:
Can we update the user doc with some very simple instructions eg "typically compression=100 is used. But the user can use higher compression for more accurate results, with higher memory consumption."

Reply:
will do once this change makes it in :)